src/common/utils/__tests__/multiYearPlanHelperFunctions.test.ts

```diff
@@ -0,0 +1,75 @@
+import { deepStrictEqual, strictEqual } from 'assert';
+import {
+  NUM_SEMESTERS,
+  NUM_YEARS,
+  TERM,
+} from 'common/constants';
+import { metadata } from 'testData';
+import flatten from 'lodash.flatten';
+import { stub } from 'sinon';
+import * as multiYearPlan from '../multiYearPlanHelperFunctions';
+import { calculateSemesters } from '../multiYearPlanHelperFunctions';
+
+describe('Multi-Year Plan Helper Functions', function () {
+  describe('calculateSemesters', function () {
+    it('returns the requested number of semesters', function () {
+      const actual = calculateSemesters(
+        metadata.currentAcademicYear, NUM_SEMESTERS
+      );
+      strictEqual(actual.length, NUM_SEMESTERS);
+    });
+    it('returns the correct terms', function () {
+      const years = Array(NUM_YEARS);
+      const terms = [TERM.FALL, TERM.SPRING];
+      const expectedTerms = flatten(years.fill(terms));
+      const actualTerms = multiYearPlan.calculateSemesters(
+        metadata.currentAcademicYear,
+        NUM_SEMESTERS
+      ).map((semester) => semester.term);
+      deepStrictEqual(actualTerms, expectedTerms);
+    });
+    it('returns the expected academic years', function () {
+      const expectedSemesterAcademicYears = [2021, 2021, 2022, 2022,
+        2023, 2023, 2024, 2024];
+      const actualAcademicYears = multiYearPlan.calculateSemesters(
+        metadata.currentAcademicYear,
+        NUM_SEMESTERS
```

Comment:
Instead of using the `NUM_SEMESTERS` constant here, I'd suggest substituting a literal `8` (or any other even number). That way if we ever do need to change the constant to make this a five-year plan, we won't break all these tests in the process.

Reply:
Good idea! I tried making each test more insular but missed this one. This was fixed by setting the number of semesters to `8` in 806778a.